engine/src/main/java/com/datatorrent/stram/client/StramAgent.java

```diff
@@ -424,7 +434,7 @@ private StramWebServicesInfo retrieveWebServicesInfo(String appId)
       return new StramWebServicesInfo(appMasterUrl, version, appPath, user, secToken, permissionsInfo);
     }
     catch (Exception ex) {
-      LOG.debug("Caught exception when retrieving web service info for app " + appId, ex);
+      LOG.error("Caught exception when retrieving web service info for app " + appId, ex);
       return null;
```

Comment:
Should we not let the caller decide what to do with the exception? CLI and gateway may have different ways to handle.